src/Event/ReadEventRestController.php

```diff
@@ -111,31 +111,27 @@ public function getCalendarSummary($cdbid, Request $request)
         $data = null;
         $response = null;
 
-        $style = ($request->query->get('style') !== null) ? $request->query->get('style') : 'text';
-        $langCode = ($request->query->get('langCode') !== null) ? $request->query->get('langCode') : 'nl_BE';
-        $hidePastDates = ($request->query->get('hidePast') !== null) ? $request->query->get('hidePast') : false;
-        $timeZone = ($request->query->get('timeZone') !== null) ? $request->query->get('timeZone') : 'Europe/Brussels';
+        $style = $request->query->get('style', 'text');
+        $langCode = $request->query->get('langCode', 'nl_BE');
+        $hidePastDates = $request->query->get('hidePast', false);
+        $timeZone = $request->query->get('timeZone', 'Europe/Brussels');
+        $format = $request->query->get('format', 'lg');
 
-        if ($request->query->get('format') !== null) {
-            $format = $request->query->get('format');
-
-            $data = $this->service->getEvent($cdbid);
-            $event = $this->serializer->deserialize($data, Event::class);
+        $data = $this->service->getEvent($cdbid);
+        $event = $this->serializer->deserialize($data, Event::class);
 
+        if ($style !== 'html' && $style !== 'text') {
```

Comment:
This could be done more elegantly, by using a switch statement. Now you have nested ifs, which increases the complexity.
default: case could be returning the api problem response.